lib/addons/aws-loadbalancer-controller/index.ts

```diff
@@ -22,7 +22,18 @@ export interface AwsLoadBalancerControllerProps extends HelmAddOnUserProps {
     /**
      * Enable WAFV2 (must be false for CN partition)
      */
-    enableWafv2?: boolean
+    enableWafv2?: boolean,
+
+    /**
+     * Create the ingressClass to be used by the ALB controller
+     */
+    createIngressClassResource?: boolean
+
+    /**
+     * Name of ingressClass to the ALB controller will satisfy. If not provided
+     * the value will be defaulted to "alb"
+     */
+    ingressClass?: string
```

Comment:
I suppose it is ignored if `createIngressClassResource` is false? We can call it out.

Reply:
This value is not ignored even if the createIngressClassResource is set to false. In that case, the helm chart will create the ALB and will expect the user to create an IngressClass with the correct name matching the ingressClass name above (default = alb). This is the same behavior as the underlying Helm chart.